### PR TITLE
refactor: Replace Typography with Text component in transaction-settings.js 

### PR DESCRIPTION
**Pull request by Devin-Apps**
Updated PR addressing comments. Link to Devin run: https://preview.devin.ai/devin/a26394cf2cd149ae97072237ac94d97d

### CODE REVIEW

**Comment by Devin-Apps** [comment]
Tagging @georgewrmarshall for review

**Review by georgewrmarshall — APPROVED** [review]
Code LGTM! It's not immediately apparent to me why the e2e tests are failing

**Review by garrettbear — APPROVED** [review]

File: ui/pages/swaps/transaction-settings/transaction-settings.js [hunk 1]
```diff
@@ -269,12 +266,12 @@ export default function TransactionSettings({
                   title={notificationTitle}
                   titleProps={{ 'data-testid': 'swaps-banner-title' }}
                 >
-                  <Typography
-                    variant={TypographyVariant.H6}
+                  <Text
+                    variant={TextVariant.bodySm}
                     testId="mm-banner-alert-notification-text"
```
Comment (georgewrmarshall) [inline ui/pages/swaps/transaction-settings/transaction-settings.js:271]: This should fix failing e2e tests

```suggestion
                    data-testid="mm-banner-alert-notification-text"
```

**Review by georgewrmarshall — COMMENTED** [review]
Left a suggestion to fix failing e2e tests

**Pull request closed without merging**